src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -81,8 +82,8 @@ public void testIndexBCFFileGZ() {
         final Object res = this.runCommandLine(args);
     }
 
-    @Test
-    public void testVCFGZIndex() {
+    @Test(expectedExceptions = UserException.class)
+    public void testVCFGZIndex_tabixRequires_tbi_name() {
         final File ORIG_FILE = new File(getToolTestDataDir(), "test_variants_for_index.vcf.blockgz.gz"); //made by bgzip
         final File outName = BaseTest.createTempFile("test_variants_for_index.blockgz.gz.", ".idx");
 
```

Comment:
Since the exception happens during the run of the tool in this test case, you should be able to delete the lines after `runCommandLine()`